docs/generate-tables.sh

```diff
@@ -1,31 +1,30 @@
 #!/bin/bash
 # Generate the documentation tables as docs/generated/OJP.html from the .xsd schema files
 #
-# You need the binary `xsltproc`
+# You need the binary `java`
 # apt-get install xsltproc
```

Comment:
this line should either be removed or adapted to `default-jre`

Reply:
I don't speak ubuntu or debian. The Java _binary_ is what is required, not its distribution package.

Reply:
I meant line 5 `apt-get install xsltproc`